### PR TITLE
[ci skip] add chinese comments

### PR DESCRIPTION
**Pull request by dabingnn**
@VisualSJ Another pull request for chinese comments, Thanks.

### CODE REVIEW

File: cocos/renderer/CCRenderCommand.h [hunk 1]
```diff
@@ -38,64 +38,154 @@
...
+    @return
+    @~english The transparent status.
+    @~chinese 透明状态。
+    */
     inline bool isTransparent() const { return _isTransparent; }
     
-    /** Set transparent flag. */
+    /** 
+    @~english Set transparent flag.  
+    @~chinese 设置透明的旗帜。
```
Comment (VisualSJ) [inline cocos/renderer/CCRenderCommand.h]: 标志？

**Comment by dabingnn** [comment]
@VisualSJ updated， Thanks

**Comment by dabingnn** [comment]
@VisualSJ pull request updated, could you please review it? Thanks.

**Comment by VisualSJ** [comment]
@andyque This PR is ok for merge.

**Pull request merged**